src/Microsoft.Data.SqlClient/tests/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj

```diff
@@ -60,6 +61,7 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp2.1' AND '$(TargetGroup)' == 'netcoreapp'">
     <PackageReference Include="System.Data.Odbc" Version="$(SystemDataOdbcVersion)" />
+    <Compile Include="SslOverTdsStreamTest.NetCoreApp.cs" />
```

Comment:
This ItemGroup applies to only netcoreapp3.1.
You can skip testing on NetFx by specifying class level tags as done in other test classes.

Reply:
The core methods are available on core 2.1 and later so yes they should probably be in another compile group but i was having a lot of trouble getting the test to compile in all configurations locally so I had to rely on the CI to show me the problems. This would only cause a problem if you ran tests against 2.1 and not 3.1 which seems an unlikely scenario. I'll see if there's a define I can change it to though.

Reply:
Oh, and this and every other project will need revisiting once we know the TFM for net 5 since it shouldn't be netcoreapp anymore